extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -195,6 +195,7 @@ class AdSenseNetworkConfig {
     return dict({
       'type': 'adsense',
       'data-ad-client': this.autoAmpAdsElement_.getAttribute('data-ad-client'),
+      'data-ad-host': this.autoAmpAdsElement_.getAttribute('data-ad-host'),
```

Comment:
Could you modify the logic so that we only add this property into the dict if the data-ad-host attribute is present on this.autoAmpAdsElement_.

The return type of this function (!JsonObject<string, string>) would imply that values can't be null. I know that it's not actually type-safe for data-ad-client, but that is a required field, so we don't expect it to ever be null, whereas we would for data-ad-host.

Reply:
How can i add the property dynamically? The linter will complain that dict needs to have an object literal expression passed. I tried passing Object.assign() since it will skip over null params, but it didn't work. I also can't pass a reference.

Reply:
@tlong2 nevermind, fixed it

Reply:
@keithwrightbos